common.py

```diff
@@ -353,24 +368,27 @@ def add_to_params(pars) :
         else :
             self.include_BAO=False
         self.n_bao=0
+        self.relative_bao_errors=False
         while config.has_section("BAO %d"%(self.n_bao+1)) :
             self.n_bao+=1
             sec_title="BAO %d"%(self.n_bao)
             if config.has_option(sec_title,'fname_da') :
                 self.include_DA=True
-                zn,eda=np.loadtxt(config.get(sec_title,'fname_da'),unpack=True)
+                zn,eda=np.loadtxt(config.get(sec_title,'fname_da'),unpack=False)
```

Comment:
I find ascii files with data in columns rather than rows easier to read. Would it be too much of a pain to revert to upack=True? The only thing you'd have to do is save the BAO data in columns rather than rows (i.e. np.savetxt("bao.txt",np.transpose(data)) rather than np.savetxt("bao.txt",data)).